doc/level1.rst

```diff
@@ -302,6 +319,7 @@ A Level 1 MUA MUST be capable of processing and handling 2048-bit and
 found in an Autocrypt header (for example, by passing it agnostically
 to an OpenPGP backend for handling).
 
+
```

Comment:
why is this extra whitespace here?

Reply:
This was also neither changed nor commented, but marked as resolved?

Reply:
Sorry... I did not notice that the push of the amended commit failed.
Now should be fine.